staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2748,6 +2748,13 @@ type LiveUpdateCPU struct {
 }
 
 type LiveUpdateConfiguration struct {
+	// MaxHotplugRatio is the ratio used to define the max amount
+	// of a hotplug resource that can be made available to a VM
+	// when the specific Max* setting is not defined (MaxCpuSockets, MaxGuest)
+	// Example: VM is configured with 512Mi of guest memory, if MaxGuest is not
+	// defined and MaxHotplugRatio is 2 then MaxGuest = 1Gi
+	// defaults to 4
+	MaxHotplugRatio uint32 `json:"maxHotplugRatio,omitempty"`
```

Comment:
In short yes :) I've just made what's already present for CPU hotplug configurable. Keep in mind this is cluster-wide, admins might want to set this instead of the fixed value to make it more flexible.